dpnp/dpnp_iface_searching.py

```diff
@@ -51,22 +49,26 @@
 __all__ = ["argmax", "argmin", "searchsorted", "where"]
 
 
-def argmax(x1, axis=None, out=None):
+def argmax(a, axis=None, out=None, *, keepdims=False):
     """
     Returns the indices of the maximum values along an axis.
 
     For full documentation refer to :obj:`numpy.argmax`.
 
+    Returns
+    -------
+    out : dpnp.ndarray
+        Indices of maximum value of `a`.
+
     Limitations
     -----------
-    Input array is supported as :obj:`dpnp.ndarray`.
-    Otherwise the function will be executed sequentially on CPU.
-    Parameter `axis` is supported only with default value ``None``.
-    Parameter `out` is supported only with default value ``None``.
+    Input and output arrays are only supported as either :class:`dpnp.ndarray`
```

Comment:
Could you please add description for input arguments and remove limitation section?